### ISSUE TITLE
Autocomplete: Screen reader doesn't read suggestions on Safari

### ISSUE THREAD

**Issue by oykuyilmaz** [ajaxorg/ace#5377]
### Describe the bug

Triggering autocomplete and navigating completions is not mentioned in screen reader on Safari.

### Expected Behavior

Autocomplete popover is accessible to screen reader users.

### Reproduction Steps

1. Open https://ace.c9.io/kitchen-sink.html
2. Start VoiceOver
3. Ctrl + Space to trigger autocomplete

### Ace Version / Browser / OS / Screen Reader

1.31.1/Safari/MacOS/VoiceOver

**Comment by oykuyilmaz** [comment]
Works as expected on Chrome.

**Comment by marinsokol5** [comment]
Can confirm that Chrome and Firefox work correctly but Safari 16.6 doesn't.  
Unsure if this is a Safari problem but we should definitely investigate if there is something we can do for those users.

**Comment by oykuyilmaz** [comment]
We use `aria-activedescendant` attribute for the current selection on autocomplete popover with `listbox` role. Screen reader support expectations are as following for this attribute:
https://a11ysupport.io/tech/aria/aria-activedescendant_attribute
